keps/sig-node/585-runtime-class/README.md

```diff
@@ -367,7 +348,7 @@ Beta:
   - [x] [dockershim](https://github.com/kubernetes/kubernetes/pull/67909)
 - [ ] Comprehensive test coverage
   - [ ] [CRI validation tests][cri-validation]
```

Comment:
I'm not sure what was intended here. @tallclair do you recall?

Reply:
The [CRI Validation tests](https://github.com/kubernetes-sigs/cri-tools/blob/master/docs/validation.md) are meant to validate that a CRI implementation conforms to the spec. However, in the case of RuntimeClass, the only relevant piece is the runtime `handler` field, which is an opaque value passed to the CRI. Since there isn't a 'spec' for the handler field, I'm not sure what would be validated here, and I can't remember what we were thinking when it was added :(

I think we can probably remove it, unless someone more involved on CRI has a better idea.

Reply:
thank you, removed